packages/siopv2-oid4vp-op-auth/src/agent/DidAuthSiopOpAuthenticator.ts

```diff
@@ -344,32 +357,62 @@ export class DidAuthSiopOpAuthenticator implements IAgentPlugin {
 
     const pex = new PEX()
     const verifiableCredentialsWithDefinition: Array<VerifiableCredentialsWithDefinition> = []
+    const dcqlCredentialsWithCredentials: Map<DcqlCredentialRepresentation, UniqueDigitalCredential> = new Map()
+
+    if (authorizationRequestData.presentationDefinitions !== undefined && authorizationRequestData.presentationDefinitions !== null) {
+      authorizationRequestData.presentationDefinitions?.forEach((presentationDefinition) => {
+        const { areRequiredCredentialsPresent, verifiableCredential: verifiableCredentials } = pex.selectFrom(
+          presentationDefinition.definition,
+          selectedCredentials.map((udc) => udc.originalVerifiableCredential!),
+        )
+
+        if (areRequiredCredentialsPresent !== Status.ERROR && verifiableCredentials) {
+          const uniqueDigitalCredentials: UniqueDigitalCredential[] = verifiableCredentials.map((vc) => {
+            // @ts-ignore FIXME Funke
```

Comment:
```suggestion
            // @ts-expect-error: Funke needs to be fixed in XYZ-1234
```

Reply:
Do we have a ticket for that? Because it's not my implementation and I have no idea what is it about.

Reply:
yeah we dropped plenty of those the night/morning before we had to deliver Funke phase 1.